src/engine.py

```diff
@@ -21,37 +31,81 @@ def enqueue(self, path: str) -> bool:
         Returns:
             bool: Was enqueueing successfull
         """
-        raise NotImplementedError()
+        if not os.path.isdir(path):
+            return False
+        
+        folder_name = os.path.basename(os.path.normpath(path))
+        output_path = os.path.join(r"D:\processed_songs", folder_name)
```

Comment:
extract `D:\processed_songs` to some kind of configuration - config file, env variable, static class with hardcoded values etc.?